openvino_tensorflow/backend.h

```diff
@@ -25,7 +26,9 @@ class Backend {
  public:
   Backend(const string& configuration_string);
   ~Backend() {
-    NGraphClusterManager::EvictAllClusters();
+    if (api::IsRewritePassEnabled()) {
+      NGraphClusterManager::EvictAllClusters();
+    }
```

Comment:
The idea behind this is to keep all clusters formed through Grappler pass, as CPU is the only possible backend for it. Since Grappler pass is triggered through an user-facing API, if a user mistakenly uses set_backend() after the Grappler pass happens, this will evict all clusters. Since Grappler is only for CPU at the moment, changing to any other backend is an invalid step (which the optimize_graph API will report and exit).

Reply:
Understood, thanks!